tests/unit/conftest.py

```diff
@@ -73,3 +74,11 @@ def httpretty_():
 
     httpretty.disable()
     httpretty.reset()
+
+
+def assert_cache_control(headers, cache_parts):
+    """Assert that all parts of the Cache-Control header are present."""
+    assert dict(headers) == Any.dict.containing({"Cache-Control": Any.string()})
+    assert (
+        headers["Cache-Control"].split(", ") == Any.list.containing(cache_parts).only()
+    )
```

Comment:
This could be made a pytest fixture and then you wouldn't need to import it, plus you could override and customize if if you ever needed to:

```python
@pytest.fixture
def assert_cache_control():
    """Return a function that asserts that all parts of the Cache-Control header are present."""

    def assert_cache_control(headers, cache_parts):
        <same function as before>

    return assert_cache_control
```

Reply:
On that day then maybe. I don't think it's bad to import it and it complicates a simple function.